tests/rados/test_upgrade_warn.py

```diff
@@ -156,6 +170,21 @@ def run(ceph_cluster, **kw):
                     log.info("Upgrade Complete...")
                     upgrade_complete = True
                     break
+            except Exception as err:
+                # checking if the Exception raised is due to bug : 2314146
+                # Error msg :  ENOTSUP: Warning: due to ceph-mgr restart, some PG states may not be up to date
+                # Module 'orchestrator' is not enabled/loaded (required by command 'orch upgrade status'):
+                # use `ceph mgr module enable orchestrator` to enable it
+                if "Module 'orchestrator' is not enabled/loaded" in err:
+                    log.info(
+                        "Intermittent issue hit. bug : 2314146. Error hit : \n %s \n",
+                        err,
+                    )
+                    continue
+                else:
+                    upgrade_complete = False
+                    log.error("Exception hit : %s", err)
```

Comment:
nit: Please change to -
`log.error("Exception hit : %s" % err)`

or

`log.error("Exception hit : ", err)`